index.js

```diff
@@ -34,7 +34,11 @@ module.exports = {
     }
     this.app = app;
 
-    let options = extend(defaultOptions, app.options['ember-bootstrap']);
+    let options = extend({}, defaultOptions, app.options['ember-bootstrap']);
+    if (process.env['BOOTSTRAPVERSION']) {
```

Comment:
I usually prefer to safeguard against this kind of thing being triggered in production, especially when it's mainly there for testing support. It's not meant to be used, but it's in code that users will see, especially the exposure in `environment.js`.